java/buildconf/ivy/ivy-suse.xml

```diff
@@ -79,6 +79,7 @@
         <dependency org="suse" name="salt-netapi-client" rev="0.20.0" />
         <dependency org="suse" name="simpleclient" rev="0.3.0" />
         <dependency org="suse" name="simpleclient_common" rev="0.3.0" />
+        <dependency org="suse" name="simpleclient_hibernate" rev="0.3.0" />
```

Comment:
Is this module only required for the tests?
In case you need it also at runtime, you need to change ant and the spec file as well to create the link and maybe add the depedency

Reply:
I've changed the configuration to add the jar at runtime.
Could you point me to the spec which should be updated?